conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -141,19 +153,24 @@ def _add_hostfactory_revoke_token(menu: ArgparseWrapper):
             title=title_formatter("Options"))
         options.add_argument('-action_type', default='revoke_token', help=argparse.SUPPRESS)
         options.add_argument('-t', '--token', metavar='VALUE', required=True,
+        #Needs TW review
                                           help='(Mandatory) the Host Factory Token to revoke')
         options.add_argument('-h', '--help', action='help',
+        #Needs TW review
                                           help='Display help screen and exit')
 
     @staticmethod
     def _add_hostfactory_create_token(menu: ArgparseWrapper):
+        #Needs TW review
         hostfactory_create_token_name = 'token - Creates a token for creating hosts ' \
                                         'with restrictions'
```

Comment:
Generate a Host Factory token for creating hosts with rrestrictions". (BTW, does the token have restrictions, or the host?)